### ISSUE TITLE
Is the siteorigin_panels_row_cell_attributes filter gone?

### ISSUE THREAD

**Issue by drumba** [siteorigin/siteorigin-panels#378]
Hey guys,

I am using your pagebuilder for about 2 years now. It helped us a lot to let our content managers build there own layouts. Thanks for that. Very nice job.

I customized the pagebuilder to use the [bootstrap grid system](http://getbootstrap.com/css/#grid). To approach this I am using the `siteorigin_panels_row_cell_attributes`, `siteorigin_panels_row_style_attributes`,  `siteorigin_panels_widget_style_attributes` filters. What I see is that the `siteorigin_panels_row_cell_attributes` filter is gone. Can you confirm that? Is it on purpose? 

With the new beta all our layouts are looking like [this](https://www.dropbox.com/s/4xj6jmiokt8td0d/Screenshot%202017-04-04%2010.40.49.png?dl=0).

**Comment by gregpriday** [comment]
Hi @drumba - well spotted! That filter was unintentionally removed. Well, I actually renamed it, but I'll add the old filter too for legacy purposes.

**Comment by gregpriday** [comment]
I readded the filter. Can you check if this fixes things for you. One thing to note, is that we changed our layouts from floats to flexbox. This might change things with your plugin - but hopefully, you're already removing all our CSS to use Bootstrap instead.

Let me know if you need me to make you a build. If you just replace `inc/renderer.php` with the version from `develop`, you should be able to check.

**Comment by LittleHamster** [comment]
@drumba did you create a bootstrap grid plugin? can you please open-source your plugin so we can use it too? thanks

**Comment by drumba** [comment]
@gregpriday Thanks for you fast change and sorry for my late reply. I just tested it and it's working perfectly. Thanks for that.

@LittleHamster It's not a standalone plugin, it's part of our core plugin and it is only supporting a subset of the bootstrap grid. For us it doesn't make sense to support every grid combination. So I guess it's not that useful for the hole community. But if you are interested how we did it let me know. I can create a gist with the important parts.
